frontend/src/components/common/input/input.tsx

```diff
@@ -30,9 +31,28 @@ const Input: FC<Props> = ({
   type = 'text',
   hasVisuallyHiddenLabel = false,
   inputClassName = '',
+  textAreaRows = 2,
```

Comment:
```suggestion
  textAreaRows,
```
we don't want to have a default params